docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -192,9 +199,9 @@ For example, to find the *Data Generator* source connector, type *data* in the s
 +
 Click the card to select the connector, and then click *Next*.
 
-. For the *Kafka instance*, click the card for the {product-kafka} instance that you configured for Connectors, and then click *Next*.
+. For the *Kafka instance*, click the card for the {product-kafka} instance that you configured for connectors, and then click *Next*.
 +
-NOTE: If you have not already configured a {product-kafka} instance for Connectors, you can create a new Kafka instance by clicking *Create Kafka instance*. You would also need to set up and define access for a service account as described in _Configuring the {product-kafka} instance for use with {product-long-connectors}_.
+NOTE: If you have not already configured a {product-kafka} instance for connectors, you can create a new Kafka instance by clicking *Create Kafka instance*. You would also need to set up and define access for a service account as described in _Configuring the {product-kafka} instance for use with {product-long-connectors}_.
```

Comment:
You can probably simplify the first sentence a little. The name of the *Create Kafka instance* button speaks for itself. Suggest:

```
NOTE: If you have not already configured a {product-kafka} instance for connectors, click *Create Kafka instance*. 
```

Reply:
For the second sentence, per IBM style, don't use "would" as an auxiliary verb. Write in the present tense instead:

"You also need to set up and define access for a service account, as described in...."